src/actions/login.js

```diff
@@ -4,6 +4,9 @@ import {goTo} from './'
 
 export const loginFromEmbedded = () => (dispatch) => {
   dispatch({type: types.LOGIN_FROM_EMBEDDED})
-  const url = `${conf.server.serviceUrl}/accounts/login/?next=${location.href}`
-  dispatch(goTo(url))
+  if (conf.organization.sso_enabled) {
```

Comment:
why is it snake case? Should be camel case.

Reply:
whatever is passed from the outside or loaded from the api should have been camelized.

Reply:
@kof updated. now config response from server camelized